### DISCUSSION TITLE
Sphinx Cross-referencing.

### DISCUSSION THREAD

**Discussion by cmichelenstrofer** [sandialabs/WecOptTool#31] [Dev]
Finding section names: https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#finding-the-reference-name 

```
python -m sphinx.ext.intersphinx _build/html/objects.inv
```

**Comment by rgcoe** [comment]
It took me a while to figure out how to do link to the API. Use command above to see list of what's available.

For some reason you use `:py:mod:` instead of `:py:module:`: https://stackoverflow.com/questions/31879104/python-module-reference-with-sphinx-documentation
